src/handler.rs

```diff
@@ -22,13 +22,37 @@ impl Handler {
         self.backtrace.as_ref()
     }
 
+    /// Return a mutable reference to the captured `Backtrace` type
+    pub fn backtrace_mut(&mut self) -> Option<&mut Backtrace> {
+        self.backtrace.as_mut()
+    }
+
+    /// Remove the captured `Backtrace` if present.
+    pub fn clear_backtrace(&mut self) {
+        self.backtrace = None;
+    }
```

Comment:
I initially tried to use the `.backtrace_mut()` as in https://github.com/yaahc/color-eyre/issues/130#issuecomment-1513771667 , and it didn't work as it returns `Option<&mut Backtrace>` rather than `&mut Option<Backtrace>`.

I _think_ that's more idiomatic rust, and so having two methods makes sense, but if you'd rather we delete `clear_backtrace()` and change the signature of backtrace_mut to:

```rust
pub fn backtrace_mut(&mut self) -> &mut Option<Backtrace> {
```

that works for me too.